models/renalStructures_UNEST_segmentation/configs/metadata.json

```diff
@@ -3,7 +3,8 @@
     "version": "0.1.0",
     "changelog": {
         "0.1.0": "complete the model package",
-        "0.0.1": "initialize the model package structure"
+        "0.1.1": "initialize the model package structure",
```

Comment:
how about just remove this line, since we do not have an actual version `0.1.1` bundle here.

Reply:
Ok, yes, this change is a bug fix. Let's remain as the same version, will delete this.

Reply:
Hi @tangy5 @yiheng-wang-nv ,

I think you should not remove the line, instead, please update the "version" to `0.1.1` and update the "changelog" to:
```
        "0.1.1": "fixed KeyError in the ChecpointLoader",
        "0.1.0": "complete the model package"
```
right?
What do you think?
Or maybe I misunderstand something?

Thanks.

Reply:
Hi @Nic-Ma , the place you saw is outdated

Reply:
Hi @Nic-Ma , 
> I think you should not remove the line
The place you saw is outdated, thus is not the place I mentioned that should be removed.

Reply:
Hi @tangy5 , 
before changing this bundle, the meta information is:
```
"version": "0.1.0",
"changelog": {
    "0.1.0": "complete the model package",
    "0.0.1": "initialize the model package structure"
},
```

Then you made some changes, thus the version number should be updated, and mentioned in the "changelog". However, there should only be one item added into the changelog. Otherwise, you need to prepare two different versions of the bundle separately.

Therefore, during this PR you can write sth like:

```
"version": "0.1.1",
"changelog": {
    "0.1.1": "what you changed should be added here",
    "0.1.0": "complete the model package",
    "0.0.1": "initialize the model package structure"
},
```

Reply:
This is very clear, thanks!
Fixed and commited